File: app/src/main/java/de/westnordost/streetcomplete/about/AboutActivity.java
package de.westnordost.streetcomplete.about;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.MenuItem;

import de.westnordost.streetcomplete.R;

public class AboutActivity extends AppCompatActivity
{
	@Override protected void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);

		setContentView(R.layout.activity_about);

		Toolbar toolbar = findViewById(R.id.toolbar);
		setSupportActionBar(toolbar);
		getSupportActionBar().setDisplayShowHomeEnabled(true);
		getSupportActionBar().setDisplayHomeAsUpEnabled(true);

		if(savedInstanceState == null)
		{
			getFragmentManager()
					.beginTransaction()
					.add(R.id.fragment_container, new AboutFragment())
					.commit();
		}
	}

	@Override
	public void onBackPressed() {
		if (getFragmentManager().getBackStackEntryCount() > 0) {
			getFragmentManager().popBackStack();
		} else {
			super.onBackPressed();
		}
	}

	// TODO consider using this instead (for UI/toolbar redesign)
	// https://stackoverflow.com/questions/29558568/manage-toolbars-navigation-and-back-button-from-fragment-in-android#29795569
	@Override
	public boolean onOptionsItemSelected(MenuItem item) {
		switch (item.getItemId()) {
			case android.R.id.home:
				onBackPressed();
				return true;
		}
		return super.onOptionsItemSelected(item);
	}
}

File: app/src/main/java/de/westnordost/streetcomplete/about/AboutActivity.kt
package de.westnordost.streetcomplete.about

import android.os.Bundle
import de.westnordost.streetcomplete.FragmentContainerActivity

class AboutActivity : FragmentContainerActivity(), AboutFragment.Listener
{
    override fun onPostCreate(savedInstanceState: Bundle?) {
        super.onPostCreate(savedInstanceState)
        if (savedInstanceState == null) {
            mainFragment = AboutFragment()
        }
    }

    override fun onClickedChangelog() {
        pushMainFragment(ChangelogFragment())
    }

    override fun onClickedCredits() {
        pushMainFragment(CreditsFragment())
    }

    override fun onClickedPrivacyStatement() {
        pushMainFragment(PrivacyStatementFragment())
    }
}
